File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[]arr=new int[2*nums.length];
        int j=0, i=0;
        while(i<2)
        {
            for(int num:nums)
            {
                arr[j++]=num;
            }
            i++;
        }
        return arr;
        
    }
}

File: Data Structures & Algorithms/combination-target-sum/submission-0.py
class Solution:
    def combinationSum(self, nums: List[int], target: int) -> List[List[int]]:
        res=[]
        def solve(i,target,ans):
            if i>=len(nums):
                if target==0:
                    res.append(ans.copy())
                return 
            if nums[i]<=target:
                ans.append(nums[i])
                solve(i,target-nums[i],ans)
                ans.pop()
            solve(i+1,target,ans)
        solve(0,target,[])
        return res